.gitlab-ci.yml

```diff
@@ -8,6 +8,7 @@ variables:
   GITHUB_REPOSITORY: ''
 
 workflow:
+  name: '$PIPELINE_NAME'
```

Comment:
What if triggered from eic_container and $PIPELINE_NAME not set?

Reply:
Then the name setting is ignored, I believe,
https://docs.gitlab.com/ee/ci/yaml/#workflowname